templates/terraform/iam_policy.go.erb

```diff
@@ -30,11 +30,12 @@ import (
 <%
 resource_uri = object.self_link_url
 resource_params = extract_identifiers(resource_uri.gsub('{{name}}', "{{#{object.name.underscore}}}"))
+ref_name = object.iam_policy.parent_resource_attribute || resource_params.last.underscore
```

Comment:
nit - it's not entirely clear what ref_name means to me, maybe field_name, resource_name or something along those lines? i have a feeling this is the same in the other templates though, so 🤷‍♀

Reply:
Renamed it to `parent_resource_name`, I think that's more clear